templates/node.json.j2

```diff
@@ -1,14 +1,11 @@
 {
-  "configuration":
-  {
-    "proxy": "{{selenium_node_proxy}}",
-    "maxSession": {{selenium_node_max_session}},
-    "port": {{selenium_node_port}},
-    "host": "{{selenium_node_host}}",
-    "register": {% if selenium_node_register %}true{% else %}false{% endif %},
-    "registerCycle": {{selenium_node_register_cycle}},
-    "hubPort": {{selenium_node_hub_port}},
-    "hubHost": "{{selenium_node_hub_host}}"
-  },
+  "proxy": "{{selenium_node_proxy}}",
```

Comment:
This needs conditional templating to support both versions.  If the version is below 3 it should wrap the settings in a configuration dictionary.